app/views/admin/tuition_fees/_form.html.erb

```diff
@@ -23,8 +23,12 @@
 
   <div class="field">
     <%= f.label :group_id %>
-    <%= f.text_field :group_id %>
+    <%= f.collection_select :group_id, Admin::TuitionFeesGroup.all,:id,:name%>
```

Comment:
Don't change this. Because I already noticed that and have made an bug issue about this. So, Please revert this.